projects/picker/src/lib/date-time/calendar.component.ts

```diff
@@ -238,6 +242,12 @@ export class OwlCalendarComponent<T> implements AfterContentInit, AfterViewCheck
   @Input()
   hideOtherMonths: boolean;
 
+  /**
+   * Flag to show today button to jump to today's date. Defaults to `false`.
+   * */
+  @Input()
```

Comment:
You could add the [`booleanAttribute`](https://angular.dev/guide/components/inputs#built-in-transformations) transformation here, which allows you to set the input just by adding this into the template:

```html
<!-- without transformation -->
<owl-date-time [showTodayButton]="true" />

<!-- with transformation -->
<owl-date-time showTodayButton="true" />
<owl-date-time showTodayButton />
```

Not required, just a nice to have.